.gitignore

```diff
@@ -1,2 +1,3 @@
 /hubble
 install/kubernetes/hubble.yaml
+*.pprof
```

Comment:
we don't need to remove it from here, but this is a fantastic addition to your global gitignore so pprof files are never checked in across all projects